### PR TITLE
Add check for docker compose plugin

### LINKED ISSUE #140: Docker Compose detection
Docker Compose has been moved to a Docker CLI plugin, so the check for a `docker-compose` executable for the book requirements: https://github.com/symfony-cli/symfony-cli/blob/48ca923fa3adc2e725788626f02501dff52ddc6a/book/reqs.go#L140 is outdated.

### PR DESCRIPTION
**Pull request by ArnaudLier**
Fixes #140

Maybe you want to check for it differently?

### CODE REVIEW

File: book/reqs.go [hunk 1]
```diff
@@ -137,7 +137,9 @@ func CheckRequirements() (bool, error) {
 	}
 
 	// Docker Compose
-	if _, err := exec.LookPath("docker-compose"); err != nil {
+	_, composeBinary := exec.LookPath("docker-compose")
+	composePlugin := exec.Command("docker", "compose").Run()
```
Comment (fabpot) [inline book/reqs.go:141]: Can't we check the version of the `docker` binary instead?

Comment (ArnaudLier) [reply book/reqs.go:141]: Presence of the docker binary does not mean you have the `compose` subcommand.

Comment (ArnaudLier) [reply book/reqs.go:141]: Oh yeah, but you can have the latest Docker CLI and no compose plugin?

Comment (tucksaun) [reply book/reqs.go:141]: Indeed. And docker compose is still available as a standalone binary as well.

Comment (fabpot) [reply book/reqs.go:141]: oh, right, this is a plugin.

**Comment by tucksaun** [comment]
The usages of `docker-compose` in `book/checkout.go` will probably needs some updates as well

**Pull request merged**